src/integration-test/resources/db/scripts/truncateErrands.sql

```diff
@@ -0,0 +1,33 @@
+set foreign_key_checks = 0;
```

Comment:
Is this file used?

Reply:
Nope, only used for debugging purposes, removed now!

Good catch!